### ISSUE TITLE
make it work again on Venus v1.73 and higher

### ISSUE THREAD

**Issue by mpvader** [izak/bms-bridge#2]
In Venus v1.73 the naming of the Battery/BMS related D-Bus service was changed. See here: 
https://git.victronenergy.com/can/lg_resu_interface/commit/e91f248689e9bd911a4d1e2a43ee9f5b8153ed6f

Some info from a v2.00 system (which is the same as v1.73 and v1.74 for this aspect of BMS D-Bus service naming):

```
root@ccgx:~# dbus -y
org.freedesktop.DBus
com.victronenergy.hub4
com.victronenergy.qwacs
com.victronenergy.system
com.victronenergy.logger
com.victronenergy.fronius
com.victronenergy.vebus.ttyO1
com.victronenergy.battery.socketcan_can0
com.victronenergy.settings
net.connman
fi.epitest.hostap.WPASupplicant
fi.w1.wpa_supplicant1
```

```
root@ccgx:~# dbus -y com.victronenergy.battery.socketcan_can0 / GetValue
value = {'Alarms/CellImbalance': 0,
 'Alarms/HighChargeCurrent': 0,
 'Alarms/HighChargeTemperature': 0,
 'Alarms/HighDischargeCurrent': 0,
 'Alarms/HighTemperature': 0,
 'Alarms/HighVoltage': 0,
 'Alarms/InternalFailure': 0,
 'Alarms/LowChargeTemperature': 0,
 'Alarms/LowTemperature': 0,
 'Alarms/LowVoltage': 0,
 'Connected': 1,
 'Dc/0/Current': 0.0,
 'Dc/0/Power': 0,
 'Dc/0/Temperature': 25.5,
 'Dc/0/Voltage': 49.069999694824219,
 'DeviceInstance': 512,
 'Info/BatteryLowVoltage': 0.0,
 'Info/MaxChargeCurrent': 110.0,
 'Info/MaxChargeVoltage': 58.099998474121094,
 'Info/MaxDischargeCurrent': 110.0,
 'Mgmt/Connection': 'CAN-bus',
 'ProductId': 45060,
 'ProductName': 'LG resu battery',
 'Redetect': 0,
 'Soc': 15,
 'Soh': 100}
```

**Comment by mpvader** [comment]
changing the D-Bus service name in to com.victronenergy.battery.socketcan_can0 on line 51 is all that's needed. Up to you @izak. I recommend to mention in the commit that that change fixes compatibility with v1.73 and later. But breaks with v1.72 and before.

That breaking is no issue. No need for anyone to run on anything before v1.74

**Comment by mpvader** [comment]
Completed in bms-bridge v1.3
